api/tests/Feature/DirectivesTest.php

```diff
@@ -142,8 +142,8 @@ public function testPluckFromObject(): void
         ]);
     }
 
-    // Ensure the canOnParent custom directive works as expected
-    public function testCanOnParent(): void
+    // Ensure canRoot directive works as expected.
+    public function testCanRoot(): void
```

Comment:
I left this test in to make sure the test works with the new directive, but it should probably be removed since we shouldn't really be testing a built-in directive of lighthouse-php.

Reply:
Good point! After making sure it passes, remove it before merging.

Reply:
- Test previously passed before removal in: https://github.com/GCTC-NTGC/gc-digital-talent/actions/runs/8117927762/job/22191137082#step:7:175 
- Test removed in: https://github.com/GCTC-NTGC/gc-digital-talent/pull/9613/commits/b1e37f5fc74e64a7d0fd0cb38d8533de05800782